### PR TITLE
Rewrite .osu file format

### LINKED ISSUE #2485: Formatting of .osu file format is not consistent
all of the properties should be formatted like the most recent update to the article

### PR DESCRIPTION
**Pull request by cl8n**
closes #2485 and adds #2705

### CODE REVIEW

**Comment by cl8n** [comment]
finally ready for review ;_ ;

File: wiki/osu!_File_Formats/Osu_(file_format)/en.md [hunk 1]
```diff
@@ -1,516 +1,362 @@
...
--   Hit Objects - A list of the beatmap's hit objects.
-
-Example:
-
-`[General]`
+| Section | Description | Content type |
+| :-- | :-- | :-- |
+| `[General]` | General information about the beatmap | `key: value` pairs |
+| `[Editor]` | Saved settings for the beatmap editor | `key: value` pairs |
+| `[Metadata]` | Information used to identify the beatmap | `key:value` pairs |
```
[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/osu!_File_Formats/Osu_(file_format)/en.md]: couldn't tell you why but the key-value spacing actually does differ per section

[the remark this answers is not in the document]
Comment (MegaApplePi) [reply wiki/osu!_File_Formats/Osu_(file_format)/en.md]: Should there be a note about this somewhere  then?

[the remark this answers is not in the document]
Comment (cl8n) [reply wiki/osu!_File_Formats/Osu_(file_format)/en.md]: idk, it's written how it is

**Pull request merged**